### PR TITLE
Add more detailed tab titles to student homework pages

### LINKED ISSUE #1863: Tab titles should reflect the subpage that is open
Currently, every single PrairieLearn page just has "PrairieLearn" as the name that shows up in tabs and window titles. This makes it so that users with multiple PrairieLearn tabs/windows open have to manually keep track of which tab has which page open. 

Here is an example of how my tabs appear to me when I do multiple pre-lectures for CS233 at once:

image

It would be nice to see the page titles contain information about the subpage that is open, formatted with information like the name of the problem that is open, the associated course, and possibly the word "PrairieLearn".

### PR DESCRIPTION
**Pull request by astrosticks**
In a page's `.js` file, overriding the value res.locals.htmlTitle now lets a page set a custom string that shows up in the browser's tab (the default is set to "Prairielearn" in `server.js`). 4b1ff82 provides a few examples where the student homework pages have richer tab titles, in order of (question number and name) (assessment) and (class). Closes #1863.

### CODE REVIEW

File: pages/studentAssessmentInstanceHomework/studentAssessmentInstanceHomework.js [hunk 1]
```diff
@@ -54,6 +54,11 @@ router.get('/', function(req, res, next) {
                 if (ERR(err, next)) return;
                 res.locals.assessment_text_templated = assessment_text_templated;
 
+                res.locals.htmlTitle = res.locals.assessment_set.abbreviation
+                  + res.locals.assessment.number
+                  + ': ' + res.locals.assessment.title
+                  + ' - ' + res.locals.course.short_name;
+
```
Comment (mwest1066) [inline pages/studentAssessmentInstanceHomework/studentAssessmentInstanceHomework.js]: I think `title` might be a bit much here? It's often quite long.

Should we indicate that these are PrairieLearn pages anywhere in the title? We could end titles with `(PrairieLearn)` or `(PL)`, for example.

File: pages/partials/head.ejs [hunk 2]
```diff
@@ -1,7 +1,8 @@
 <meta charset="utf-8">
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<title>PrairieLearn</title>
+<!-- <title>PrairieLearn</title> -->
```
Comment (mwest1066) [inline pages/partials/head.ejs]: Feel free to delete this.

**Comment by mwest1066** [comment]
See #1863

**Comment by mwest1066** [comment]
Is there an update on this PR? I think it only needs minimal changes before merging.

**Comment by astrosticks** [comment]
I made a more elegant solution for this on another branch, so I'll create a new pull request for it.

**Pull request closed without merging**